docs-requirements.txt

```diff
@@ -1,10 +0,0 @@
-docutils>=0.15.2,<0.17
```

Comment:
Makes sense to delete this since we now install the `dev` target in the read the docs yaml